ansys/mapdl/core/mapdl.py

```diff
@@ -2222,6 +2224,10 @@ def run(self, command, write_to_log=True, **kwargs):
             return self.list(*command.split(",")[1:])
 
         text = self._run(command, **kwargs)
+
+        if kwargs['mute']:
+            return
```

Comment:
This is a little more efficient. If the key isn't there, then None will be returned. Now, we don't need `setdefault`.
```suggestion
        if kwargs.get('mute'):
            return
```

Reply:
Very useful suggestion. Thank you!